IsotropicTextureMat.C

```diff
@@ -50,7 +50,7 @@ void IsotropicTextureMat::parse (const TiXmlElement* elem)
     utl::getAttribute(child,"min",minmax.first);
     utl::getAttribute(child,"max",minmax.second);
 
-    materials.insert(std::make_pair(minmax,LinIsotropic()));
+    materials.insert(std::make_pair(minmax,LinIsotropic(!planeStress,axiSymmetry)));
```

Comment:
This looks very strange. Why do you want to negate the planeStress option here? Like, if you specify plane stress from outside, each of the sub-material become plane strain, and vice-versa. If that is intended you have some explanation to do. if the `` was included by accident, it should be removed before merge.

Reply:
indeed i think this is wrong (my doing - too much copy-paste)